cypress/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["plugin:cypress/recommended"]
```

Comment:
I've added this because my linter was highlighting everything in red in my IDE 😄